SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -41,18 +45,29 @@ void SpatialRPCService::ProcessChanges(const float NetDriverTime)
 	ClientServerRPCs.ProcessChanges();
 	MulticastRPCs.ProcessChanges();
 
-	if (NetDriverTime - LastProcessingTime > GetDefault<USpatialGDKSettings>()->QueuedIncomingRPCRetryTime)
+	if (NetDriverTime - LastIncomingProcessingTime > GetDefault<USpatialGDKSettings>()->QueuedIncomingRPCRetryTime)
 	{
-		LastProcessingTime = NetDriverTime;
+		LastIncomingProcessingTime = NetDriverTime;
 		ProcessIncomingRPCs();
 	}
+
+	if (NetDriverTime - LastOutgoingProcessingTime > GetDefault<USpatialGDKSettings>()->QueuedOutgoingRPCRetryTime)
+	{
+		LastOutgoingProcessingTime = NetDriverTime;
+		ProcessOutgoingRPCs();
```

Comment:
Why was this added? We have the incoming one because they may have been queued, but I'm not aware of a reason for queueing outgoing rpcs.

Reply:
Added is a strong word- this is moved, and only looks different because of how we now choose to process the outgoings. (i.e. uses delta time rather than a timermanager). This logic is the same as it was in the Sender. Are you saying we can remove it?

Reply:
Ahh we gotta queue rpcs while entity creation is in progress. I guess that means we have to stick this around. I'm not sure why this is on a timer though, I thought it was just flushed at the end of every frame.

Reply:
we could change to be flushed every frame I guess- also doesn't this mean we could flush before the entity creation is complete right now, if this is only on a timer? Are we hoping creations come back before next flush?

Reply:
I believe we attempt to flush whenever we try to send an rpc currently, which might not be every frame, but most likely is.

Reply:
I think we changed the commands one to flush every frame and can probably do the same here. Currently we flush every frame by coincidence which isn't great. As when you don't flush every tick it doesn't do what you expect.

Reply:
Although the rpc refactor etc will also do this.